backend/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -114,7 +110,32 @@ public void initCounters() {
                 .onFailure().invoke(failure -> LOGGER.warnf("RBAC S2S call failed: %s", failure.getMessage()));
     }
 
-    private Uni<List<User>> getWithPagination(Function<Integer, Uni<Page<RbacUser>>> fetcher) {
+    private Uni<List<User>> getWithPagination(Uni<List<ITUserResponse>> itUserResponses) {
+        return itUserResponses.onItem().transform(page -> page.stream().map(itUserResponse -> {
+                    User user = new User();
+//                    user.setUsername(itUserResponse.getPersonalInformation().getUsername());
+                    user.setEmail(itUserResponse.getAccountRelationships().get(0).getEmails().toString());
+//                    user.setAdmin(rbacUser.getOrgAdmin());
+//                    user.setActive(rbacUser.getActive());
```

Comment:
I understand that this is still a draft, but does the IT service allows to know the username / admin / active status?

Reply:
admin, yes. username and active status I don't know yet

Reply:
username is probably principal